From 39e244a4b5b25120bcb2bb836f18e1a886cd5579 Mon Sep 17 00:00:00 2001
From: ChristofKaufmann <christofkaufmann.dev@gmail.com>
Date: Tue, 18 Apr 2023 23:44:51 +0200
Subject: [PATCH] Add missing phys2d asset in setup

In the installed package the `assets/` directory is missing in `envs/phys2d/`, because it is not listed at `tool.setuptools.package-data` in `pyproject.toml`. This commit fixes it.
---
 pyproject.toml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pyproject.toml b/pyproject.toml
index bab7890b..6674b881 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -103,6 +103,7 @@ include = ["gymnasium", "gymnasium.*"]
 gymnasium = [
     "envs/mujoco/assets/*.xml",
     "envs/classic_control/assets/*.png",
+    "envs/phys2d/assets/*.png",
     "envs/toy_text/font/*.ttf",
     "envs/toy_text/img/*.png",
     "py.typed",
-- 
2.40.0

